source/SIL.AppBuilder.Portal.Frontend/src/data/live.tsx

```diff
@@ -34,21 +34,21 @@ const DataContext = React.createContext<ILiveDataContext>({
 export function useLiveData(subscribeTo?: string) {
   const dataCtx = useContext<ILiveDataContext>(DataContext);
   const {
-    socket: { connection },
+    socket: { hub },
     isConnected,
   } = dataCtx;
 
-  const { isSubscribed } = useSubscribeToResource(connection, subscribeTo, isConnected);
+  const { isSubscribed } = useSubscribeToResource(hub, subscribeTo, isConnected);
 
   const pushData = useCallback(
     (transforms: TransformOrOperations) => {
       if (isTesting) {
         console.debug('testing early return: figure out how to test websocket communication');
         return;
       }
-      if (!isConnected) {
```

Comment:
this was just testing what would happen.
It appears that isConnected is getting set _late_. The socket appears to sometimes be connected before this variable's value becomse true